src/chain.h

```diff
@@ -23,6 +23,7 @@ static const int SPROUT_VALUE_VERSION = 1001400;
 static const int SAPLING_VALUE_VERSION = 1010100;
 static const int CHAIN_HISTORY_ROOT_VERSION = 2010200;
 static const int NU5_DATA_VERSION = 4050000;
+static const int TRANSPARENT_VALUE_VERSION = 5040000;
```

Comment:
This was not present in v5.4.0-rc1, so the earliest this should be set to is v5.4.0-rc2:
```suggestion
static const int TRANSPARENT_VALUE_VERSION = 5040026;
```